### PR TITLE
Add API for fetching transactions from all accounts

### PR DESCRIPTION
**Pull request by nathanielmiller20**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,6 +277,42 @@ client.links.getInfo({ url: "https://www.jet.com" })
   .catch(handleError);
 ```
 
+### Transactions
```
Comment (griffinmyers) [inline README.md]: Reading this documentation I've be confused about the "accounts" scoped transactions and this top-level resource. Wdyt about adding a bit of prose here describing when you'd use either?

File: README.md [hunk 2]
```diff
@@ -276,6 +277,42 @@ client.links.getInfo({ url: "https://www.jet.com" })
...
+Transactions are a paged resource.  The response object will contain properties `meta.next` and `meta.previous` which can be supplied to subsequent invocations of `#all` to fetch additional results.
+
+`#all` accepts an optional second parameter, `options` which may define the follow keys to narrow results:
+
+###### options
+
+* `cursor`: An API cursor to fetch a specific set of results
+* `start`: An ISO-8601 datetime string to filter only transactions after `start`
+* `end`: An ISO-8601 datetime string to filter only transactions before `end`
+* `time_field`: Time field to filter by: created_date (default) or modified_date
```
Comment (griffinmyers) [inline README.md]: Recommend make the relationship of this field to `start` and `end` explicit.

File: README.md (hunk 1 above)
Comment (nathanielmiller20) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (nathanielmiller20) [reply README.md]: Done

**Pull request merged**